pkgs/by-name/hi/hiddify-app/package.nix

```diff
@@ -11,41 +11,41 @@
   autoPatchelfHook,
 }:
 let
-  pname = "hiddify-app";
-  version = "2.5.7-unstable-2024-11-18";
-  src = fetchFromGitHub {
-    owner = "hiddify";
-    repo = "hiddify-app";
-    rev = "9d3de0ae2ea2687f189f75fa2a9196a035a0bb32";
-    hash = "sha256-P04A14lFfgvl0kkxMsNXNaHnwfJ3AWkhrfI7VMESGHc=";
-    fetchSubmodules = true;
+  metaCommon = {
+    description = "Multi-platform auto-proxy client, supporting Sing-box, X-ray, TUIC, Hysteria, Reality, Trojan, SSH etc";
+    license = with lib.licenses; [
+      unfree
+      gpl3Only
+    ];
```

Comment:
Please explain the situation at least a little here; this is not typical and not really a sensible combination of licenses.

Reply:
Sorry, I was unclear: Explain it in a code comment. It's not clear when looking at the code.